wagtail/contrib/redirects/utils.py

```diff
@@ -45,5 +47,10 @@ def get_file_storage():
     raise Exception("Invalid file storage, must be either 'tmp_file' or 'cache'")
 
 
+def get_redirect_hash(url):
+    namespace = "4ccf35b6-0e63-49ec-a878-50e680f3ecfc"
+    return uuid.uuid5(uuid.UUID(namespace), url)
```

Comment:
Question: Is there a reason to use a UUID rather than a literal hash? MD5 is used for similar things across Wagtail, and it seems unnecessary to use a UUID for this instead.

Reply:
I believe this was based on a recommendation I made on the original PR, as it's the same as the approach taken for the reference index (which is the most recent thing)

Reply:
Indeed, this was based on Andy's recommendation.